Index: app/src/main/java/com/example/quickcash/CredentialsValidator.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.quickcash;\r\n\r\npublic class CredentialsValidator {\r\n\r\n    public static boolean isEmptyInput(String Input) {\r\n        return (Input.matches(new String()));\r\n    }\r\n\r\n    public static boolean isValidName(String Name) {\r\n        //Do something\r\n        return true;\r\n    }\r\n\r\n    public static boolean isValidEmailAddress(String emailAddress){\r\n        return (emailAddress.matches(\"[[a-zA-Z]*[0-9]*]+@[a-zA-Z]+\\\\.[a-zA-Z]+\"));\r\n    }\r\n\r\n    public static boolean isValidRole(String userRole) {\r\n        return (userRole.equals(\"Employer\") || userRole.equals(\"Employee\"));\r\n    }\r\n\r\n    public static boolean isValidPassword(String Password, String Name, String emailAddress) {\r\n        //Do something\r\n        return false;\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/quickcash/CredentialsValidator.java b/app/src/main/java/com/example/quickcash/CredentialsValidator.java
--- a/app/src/main/java/com/example/quickcash/CredentialsValidator.java	(revision 9ed0c5447a601bfcdab4f35383b5c09345140c1a)
+++ b/app/src/main/java/com/example/quickcash/CredentialsValidator.java	(date 1728319251591)
@@ -20,7 +20,6 @@
     }
 
     public static boolean isValidPassword(String Password, String Name, String emailAddress) {
-        //Do something
-        return false;
+        return ((!Password.contains(Name) && !Password.contains(emailAddress)) && Password.length() >= 8);
     }
 }
